test/integration/attachment_deletion_integration_test.rb

```diff
@@ -14,7 +14,6 @@ class AttachmentDeletionIntegrationTest < ActionDispatch::IntegrationTest
       let(:first_asset_id) { "asset_manager_id" }
       let(:second_attachment) { build(:file_attachment, attachable: edition) }
       let(:second_asset_id_original) { "asset_manager_id_original" }
-      let(:second_asset_id_thumbnail) { "asset_manager_id_thumbnail" }
```

Comment:
Just as a catch-all comment on all these integration tests, they need love, they've always needed love. I should hope they will get some love when we revisit them in the bug fix card.